src/Resources/public/js/app/components/braintree-method-component.js

```diff
@@ -33,10 +33,10 @@ define(function (require) {
         nonceInput: null,
 
         initialize: function (options) {
-            this.options = _.extend({}, this.options, options);
```

Comment:
Where did underscore go? Was it removed in a previous commit and these usages were missed?

Reply:
Ahh cool cool, at least the code is improving!